app/views/doorkeeper/authorized_applications/index.html.erb

```diff
@@ -5,54 +5,56 @@
   <div class="col-md-9">
     <h1>
       My Applications
-    </h1>
-    <h2>
-      Developer applications
       <div class="pull-right">
         <%= link_to 'New Application', new_oauth_application_path, class: 'btn btn-success' %>
```

Comment:
I think this should be renamed to something like "New Developer Application" or "Create Application".  Otherwise I could see users who don't have applications get confused thinking they could add a new gov application to their account.